code/datums/components/tippable.dm

```diff
@@ -16,6 +16,8 @@
 	var/datum/callback/post_tipped_callback
 	/// Callback to additional behavior after being untipped.
 	var/datum/callback/post_untipped_callback
+	///The timer given until the borg untips themselves
+	var/self_untip_timer
```

Comment:
this component isn't only applied to borgs, pls

Reply:
stupid cows and medbots